TSPL.docc/ReferenceManual/Declarations.md

```diff
@@ -1465,13 +1465,25 @@ func <#function name#>(<#parameters#>) throws -> <#return type#> {
 }
 ```
 
+A function that throws a specific error type has the following form:
+
+```swift
+func <#function name#>(<#parameters#>) throws(<#error type#>) -> <#return type#> {
+   <#statements#>
+}
+```
+
 Calls to a throwing function or method must be wrapped in a `try` or `try!` expression
 (that is, in the scope of a `try` or `try!` operator).
 
-The `throws` keyword is part of a function's type,
-and nonthrowing functions are subtypes of throwing functions.
-As a result, you can use a nonthrowing function
+A function's type includes whether it can throw an error,
+and what type of error it throws.
+This means, for example, you can use a nonthrowing function
```

Comment:
The example given here is representative, not exhaustive — and the old wording is somewhat misleading, in suggesting this is the only consequence of being a subtype.  There are other things that you can do with a subtype besides just using it where the supertype is expected.  With that in mind, do you think "so" is an improvement here?  I don't think it matches the meaning.

Reply:
OK to leave the as is if you don't think the original is too misleading. I was thinking of "So" as a shorter way to say "As a result," which is what I thought "This means" was conveying here.

I also don't love using "this" (by itself) as a sentence subject because there's always the risk it's unclear, especially to someone for whom English is a second language. But I'm not going to die on that hill. Just explaining my suggestion.

Reply:
Rewrote to avoid the bare "this" which we do try to avoid elsewhere in TSPL, per DevPubs style.